api/src/transactions.rs

```diff
@@ -508,99 +562,192 @@ impl TransactionsApi {
         }
     }
 
-    async fn create(
+    fn get_signed_transactions_batch(
+        &self,
+        ledger_info: &LedgerInfo,
+        data: SubmitTransactionsBatchPost,
+    ) -> Result<Vec<SignedTransaction>, SubmitTransactionError> {
+        match data {
+            SubmitTransactionsBatchPost::Bcs(data) => {
+                let signed_transaction = bcs::from_bytes(&data.0)
+                    .context("Failed to deserialize input into SignedTransaction")
+                    .map_err(|err| {
+                        SubmitTransactionError::bad_request_with_code(
+                            err,
+                            AptosErrorCode::InvalidInput,
+                            ledger_info,
+                        )
+                    })?;
+                Ok(signed_transaction)
```

Comment:
This should say `signed_transactions` I looked at it several times and was confused.